### DISCUSSION TITLE
Welcome to the preservation effort!

### DISCUSSION THREAD

**Discussion by xpollen8** [xpollen8/jazzbutcher.com#45] [Announcements]
Hello!

I'm David Whittemore.  Like many of you, the JBC has been important to me and being in Pat's sphere of influence has colored my life to a certain extent.  I would not be sharing this had I not learned about programming for the web way back. 

I'm glad you're here!

If you're not already a collaborator, drop your request for an invite below.

**Comment by wnerini** [comment]
I'm here for you. Just send an invite.

**Reply by xpollen8** [reply]
Hey Will. Thanks. 

 Now then, about your ECC thumb pics..
                                                       
What's your relationship to Gunderson?  (He's been pretty quiet recently)

**Reply by hischeapmoves** [reply]
This is not where I expected to run into the ECC, but I am definitely here for it.

**Reply by wnerini** [reply]
HAHA! No direct connection (though I have a buddy who's  a buddy of Gunderson's.) Me, I'm just an old lover of weird music, came to the ECC through both Rudy Schwartz and Negativland.  

I realized I didn't have a pic for my avatar on GitHub, and the "Rocked By Rape" cover was the first one on my drive that had a cat on it! LMAO.

**Reply by JohnnyNebulous** [reply]
Hello!

**Reply by xpollen8** [reply]
Welcome, Johnny. Lemme know if you have questions.

**Comment by xpollen8** [comment]
This party is just getting set up (thanks for joining!)

I've dropped a couple things into the Wiki, and added a few "TODOs" to the Project.

Please be extra chatty, as I've not managed a github project before, and things can seem a bit scattered.

If you can improve things/make things clearer for the next people through the door, have at it.

-----

Right now, it seems that just completing some TODOs/chasing down archivable material is the low fruit.

Eventually, we'll get into real coding and whatnot.

**Reply by Bigquestionsinc** [reply]
James Duval, here! Count me in!!

**Reply by aaroach** [reply]
Hi there @xpollen8  - I've grabbed the mp3 and the summary transcript for the Stereo Embers Podcast. I'll await your instructions on how to upload to S3 bucket (or however you want the assets uploaded).

**Reply by xpollen8** [reply]
Go ahead and attatch the transcript to the Issue if it makes sense to.

If you have a Dropbox or something similar, let's use that for now to pass stuff around.

Thanks!

**Reply by aaroach** [reply]
Thanks  - attached the transcript to the issue thread. The mp3 I'll need to set up a sharing site. I'll do that in the next day.

**Reply by aaroach** [reply]
OK @xpollen8  - I put 2 links in the issue thread (Google Drive or Dropbox). Figured I'd try 2 different methods. Same files in each. Let me know which works better for you and I'll use that for future assets. Or, if neither works, I'll figure something else out... :-)

**Reply by xpollen8** [reply]
TY, sir. I'll give it a look in a bit. I'm working on coding up a custom file uploader for us all to use that'll dump right into S3.

**Reply by hischeapmoves** [reply]
I have some GitHub experience, and may be able to answer questions. So far, everything looks good.

**Comment by aaroach** [comment]
Andy Roach here - I'm in too. Please send an invite when you can. Thanks for asking for contributors!

**Comment by MrDithers** [comment]
Count me in. I'm a noob to GitHub, but I figure the least I can do is trawl for pics and help spread the word

**Comment by JohnnyNebulous** [comment]
Another GitHub noob here!

**Reply by mitchell-dickerman** [reply]
Happy to help, please send an invite.

**Comment by presto211** [comment]
Knight Berman here. Not sure how this all works, but happy to help if I can! Thanks, David.

**Reply by xpollen8** [reply]
Hey Knight, do you have any remaining video from the tours that hasn't been shared yet?  Or was that Scott Schleentz's thing?

**Comment by djbuege** [comment]
Please add me.  I have some bootlegs I recorded from the late 80s I'd like to share, in tribute to to Pat and all.  All from America's Midwest.  Now I'm trying to turn my 8th graders on to the JBC.

**Reply by xpollen8** [reply]
Howdy.  There is a custom uploader for collaborators to share materials such as your describing.

upload.jazzbutcher.com

I request that in addition to the audio file, please include a .txt file that includes as much information as possible about the item being uploaded: who/where, generation, equipment, and attribution to be used on the JBC site.

Thanks, and welcome!

**Reply by xpollen8** [reply]
I appreciate everything you've provided to date, and it will be exposed as I get to them!

**Comment by presto211** [comment]
That’s Scott, I’m sure. Do you want me to contact him about it (I’m overdue for catching up with him anyway)?~KBJrSent from my iTARDIS phonehttp://about.me/bermanOn Feb 22, 2023, at 4:26 PM, xpollen8 ***@***.***> wrote:﻿
Hey Knight, do you have any remaining video from the tours that hasn't been shared yet?  Or was that Scott Schleentz's thing?
